File: ce/snippets/csharp/CRMV8/optionsets/cs/workwithglobaloptionsets11.cs
// <snippetworkwithglobaloptionsets11>


 // Use the DeleteOptionValueRequest message 
 // to remove the newly inserted label.
 DeleteOptionValueRequest deleteOptionValueRequest =
     new DeleteOptionValueRequest
 {
     OptionSetName = _globalOptionSetName,
     Value = _insertedOptionValue
 };

 // Execute the request.
 _serviceProxy.Execute(deleteOptionValueRequest);

// </snippetworkwithglobaloptionsets11>

File: ce/customerengagement/on-premises/snippets/visualbasic/CRMV8/optionsetsvb/vb/workwithglobaloptionsets11.vb
' <snippetworkwithglobaloptionsets11>


 ' Use the DeleteOptionValueRequest message 
 ' to remove the newly inserted label.
 Dim deleteOptionValueRequest As DeleteOptionValueRequest =
  New DeleteOptionValueRequest With {
   .OptionSetName = _globalOptionSetName,
   .Value = _insertedOptionValue
  }

 ' Execute the request.
 _serviceProxy.Execute(deleteOptionValueRequest)

' </snippetworkwithglobaloptionsets11>